sky/data/mounting_utils.py

```diff
@@ -60,7 +60,7 @@ def get_mounting_command(
           # Check if mount path contains files
           if [ "$(ls -A $MOUNT_PATH)" ]; then
             echo "Mount path $MOUNT_PATH is not empty. Please make sure its empty."
```

Comment:
```suggestion
            echo "Mount path $MOUNT_PATH is not empty. Please mount to another path or remove it first."
```
(Suggest a fix for common scenarios first; e.g., if people accidentally mount to /tmp, the first fix should be use another path)

--- 

NOTE: I find it confusing that we allow mounting to a `/empty-dir` without files. The convention in other tools seems to be if the path exists, regardless of whether files exist, then back off.

Reply:
Updated the comment.

What kind of potential issues can be arised by allowing to mount on existing directories?